app/controllers/api/account.php

```diff
@@ -2200,6 +2200,10 @@
         if (empty(App::getEnv('_APP_SMTP_HOST'))) {
             throw new Exception(Exception::GENERAL_SMTP_DISABLED, 'SMTP Disabled');
         }
+        
+        if($user->getAttribute('emailVerification') == true){
```

Comment:
Use a linter. You're missing a whitespace after the `if` keyword.
```suggestion
        if ($user->getAttribute('emailVerification') == true) {
```

Reply:
done